packages/wallet/src/redux/actions.ts

```diff
@@ -53,6 +53,15 @@ export interface DisplayMessageSent {
   type: "WALLET.DISPLAY_MESSAGE_SENT";
 }
 
+export interface AssetTransferredEvent {
+  type: "WALLET.ASSET_HOLDER.ASSET_TRANSFERRED";
+
+  // This is either the `channelId` (bytes32) or (the left-padded bytes20)
+  // recipient address
```

Comment:
I think we should use the term "external address" which is the official name from the docs

Reply:
It is - will update